doc/playbook.rst

```diff
@@ -310,7 +309,7 @@ Install the Prompt Processing code, and set it up before use:
     setup -r prompt_processing
 
 The tester scripts send ``next_visit`` events for each detector via Kafka on the ``next-visit-topic`` topic.
-They then upload a batch of files representing the snaps of the visit to the ``rubin:rubin-pp`` S3 bucket, simulating incoming raw images.
+They then upload a batch of files representing the snaps of the visit to the ``rubin-pp-dev`` S3 bucket, simulating incoming raw images.
 
 Eventually a set of parallel processes running on multiple nodes will be needed to upload the images sufficiently rapidly.
```

Comment:
We already have this! 😁 (Well, we didn't need the "multiple nodes" part.)

Reply:
👍  . Update this in a [separate commit ](https://github.com/lsst-dm/prompt_processing/pull/120/commits/a50fe44385d7e0003b506af0db16d0c891c71d03)